strings/tips.txt

```diff
@@ -18,12 +18,32 @@ Maintenance is full of equipment that is randomized every round. Look around and
 Some roles cannot be antagonists by default, but antag selection is decided first. For instance, you can set Security Officer to High without affecting your chances of becoming an antag -- the game will just select a different role.
 There are many places around the station to hide contraband. A few for starters: linen boxes, toilet cisterns, body bags. Experiment to find more!
 You can use a machine in the vault to deposit cash or rob Cargo's department funds.
+When in doubt, clear your cache (byond launcher > cogwheel > preferences > game prefs), update your BYOND, and relog.
```

Comment:
While it should be clear enough, maybe preface by noting "When in doubt about technical issues,"